docs/changelog/CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent/): support `m` unit for `minutes` duration in command-line flag `-streamAggr.dedupInterval`. Previously unit `m` wasn't supported correctly.
 * BUGFIX: [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): properly apply replication factor when storage node groups are used and replication factor is configured via global value such as `-replicationFactor=2`. Previously, global replication factor was ignored for storage node groups. See [these docs](https://docs.victoriametrics.com/cluster-victoriametrics/#vmstorage-groups-at-vmselect) for more information about storage groups configuration.
 * BUGFIX: `vmselect` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): properly process response in [multi-level cluster setup](https://docs.victoriametrics.com/cluster-victoriametrics/#multi-level-cluster-setup). Before, vmselect could return no data in multi-level setup. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/7270) for details. The issue was introduced in [v1.104.0](https://docs.victoriametrics.com/changelog/#v11040).
-* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): fix blocking hot-reload process if the old rule group hasn't started successfully.
+* BUGFIX: [vmalert](https://docs.victoriametrics.com/vmalert): properly apply configuration changes during hot-reload to rule groups that haven't started yet. Previously, configuration updates to such groups could have resulted into blocking all evaluations within the group, until vmalert restart.
```

Comment:
>Previously, configuration updates to such groups could have resulted into blocking all evaluations within the group, until vmalert restart.

@hagen1778 not true, only reload process is blocked by this bug, and everything backs to normal automatically when group randSleep is over.
Previously, when the group is sleeping, the `Group.UpdateWith` is blocked, making the current reload blocking as well, along with any future reload calls.
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/1cb32ee6c8166f28e2fa632d93d0b1357ad7fb22/app/vmalert/rule/group.go#L453